### PR TITLE
fix(internals): remove libquery_engine from npm package

### LINKED ISSUE #15244: Prisma 4.3.1 distribution is bundling a libquery_engine
### Bug description

The prisma distribution is bundling the debian libquery engine which results in larger runtime builds and can cause issues when deploying to lambda via zip.

```
ls node_modules/@prisma/internals/dist/libquery_engine-*                                                                                                                                
node_modules/@prisma/internals/dist/libquery_engine-debian-openssl-1.1.x.so.node
```

### How to reproduce

1. `yarn add @prisma/client` or `npm install @prisma/client`
2. ls node_modules/@prisma/internals/dist/libquery_engine-*

### Expected behavior

The libquery_engine for debian or any OS for that matter would not be within the `internals/dist`.

### Prisma information

Not relevant.

### Environment & setup

- OS: macOS
- Database: PostgreSQL
- Node.js version: v16.17.0

### Prisma Version

```
Environment variables loaded from .env
prisma                  : 4.3.1
@prisma/client          : 4.3.1
Current platform        : darwin
Query Engine (Node-API) : libquery-engine c875e43600dfe042452e0b868f7a48b817b9640b (at node_modules/@prisma/engines/libquery_engine-darwin.dylib.node)
Migration Engine        : migration-engine-cli c875e43600dfe042452e0b868f7a48b817b9640b (at node_modules/@prisma/engines/migration-engine-darwin)
Introspection Engine    : introspection-core c875e43600dfe042452e0b868f7a48b817b9640b (at node_modules/@prisma/engines/introspection-engine-darwin)
Format Binary           : prisma-fmt c875e43600dfe042452e0b868f7a48b817b9640b (at node_modules/@prisma/engines/prisma-fmt-darwin)
Format Wasm             : @prisma/prisma-fmt-wasm 4.3.0-32.c875e43600dfe042452e0b868f7a48b817b9640b
Default Engines Hash    : c875e43600dfe042452e0b868f7a48b817b9640b
Studio                  : 0.473.0
```

### PR DESCRIPTION
**Pull request by Jolg42**
Closes https://github.com/prisma/prisma/issues/15244
Related https://github.com/prisma/prisma/issues/14141

Before
https://www.npmjs.com/package/@prisma/internals Unpacked Size 124 MB

After, from `pnpm publish --dry-run --no-git-checks`
npm notice package size:  77.7 kB
npm notice unpacked size: 397.7 kB

### CODE REVIEW

File: packages/internals/package.json [hunk 1]
```diff
@@ -22,6 +22,8 @@
   "files": [
     "README.md",
     "dist",
+    "!**/libquery_engine*",
+    "!dist/get-generators/engines/*",
```
Comment (jkomyno) [inline packages/internals/package.json:26]: Wouldn't just `!dist/get-generators/engines/*` be enough?

Comment (Jolg42) [reply packages/internals/package.json:26]: On my local computer, I have these, for example 😅 
```
npm notice 33.2MB dist/libquery_engine-darwin-arm64.dylib.node 
npm notice 33.1MB dist/get-generators/engines/1545b76828e10ec7b4630c6ef889b07e389d12b2/libquery_engine-darwin-arm64.dylib.node
npm notice 33.2MB dist/get-generators/engines/39190b250ebc338586e25e6da45e5e783bc8a635/libquery_engine-darwin-arm64.dylib.node                                                      
npm notice 33.2MB dist/get-generators/libquery_engine-darwin-arm64.dylib.node   
```

`get-generators` it's only used for tests and can be completely

**Review by millsp — APPROVED** [review]

**Comment by Jolg42** [comment]
4.7.1
```
npm info @prisma/internals
dist
.tarball: https://registry.npmjs.org/@prisma/internals/-/internals-4.7.1.tgz
.shasum: 7eb4c08572b1f58fee1d9ddbc00119711542af90
.integrity: sha512-YgG+5zLZuKdVUPnNdw9XA0XGfW1vZaOqrcGgRnla0mSIIIB3Gg3noRuHv9ZQAXeHLWh2v7MHxal6fxKElMN0ug==
.unpackedSize: 124.4 MB
```

Released in internal dev `4.8.0-dev.24`
```
npm info @prisma/internals@4.8.0-dev.24
dist
.tarball: https://registry.npmjs.org/@prisma/internals/-/internals-4.8.0-dev.24.tgz
.shasum: 1eaea98bfcc1702380beb41373c6ce459377d4d9
.integrity: sha512-1d7+CZnyqG2a5iXdJg1UD504MhPExzbeKvcU8Kqb/lePkyXV76ay2urzTpEjq2ZKHbune/GZq6PuXtsaY7gDrQ==
.unpackedSize: 395.4 kB
```

**Comment by Jolg42** [comment]
And ecosystem-tests are passing, like expected
https://github.com/prisma/ecosystem-tests/actions/runs/3632197520
https://github.com/prisma/ecosystem-tests/actions/runs/3632197507

**Pull request merged**